packages/cli/openapi-ir-to-fern/src/buildWebhooks.ts

```diff
@@ -27,6 +28,13 @@ export function buildWebhooks(context: OpenApiIrConverterContext): void {
                 schema: webhook.payload,
                 context,
                 fileContainingReference: webhookLocation.file
+            }),
+            examples: webhook.examples.map((exampleWebhookCall) => {
+                return {
+                    docs: exampleWebhookCall.description,
+                    name: exampleWebhookCall.name,
+                    payload: convertFullExample(exampleWebhookCall.payload)
+                };
```

Comment:
nit: can we keep examples undefined if the list is length 0 (keeps the fern definition as concise as possible)